### DISCUSSION TITLE
How to specify php version with frankenphp

### DISCUSSION THREAD

**Discussion by Rom1906** [dunglas/symfony-docker#568] [General]
Hello on build is it possible to specify php version ? To choose between 8.2 or 8.3 
Start of my Dockerfile : 
`
FROM dunglas/frankenphp:latest-alpine AS frankenphp_upstream`
Thanks !

**Comment by maxhelias** [comment]
Choose the tag that suits you best:
- PHP 8.2 : https://hub.docker.com/layers/dunglas/frankenphp/1-php8.2-alpine/images/sha256-876a3df0199278c65192e5e179ff38d23de79518d1b8364106783dcf22445feb?context=explore
- PHP 8.3 : https://hub.docker.com/layers/dunglas/frankenphp/1-php8.3-alpine/images/sha256-3e6b04fd809c9fcc573a1705e102025c09b1aed4a16ee0b4508234e45e99e1ec?context=explore
